spec/models/user_spec.rb

```diff
@@ -1,5 +1,32 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'Validaiton tests for user' do
+    subject { User.new(name: 'Henry Kc', photo: 'profile.png', bio: 'Henry the great', posts_counter: 0) }
+    before { subject.save }
+
```

Comment:
Please add unit test for `#last_recent_posts` of User model. It's a requirement to test all custom methods of a model.
![rq](https://user-images.githubusercontent.com/35504149/151603740-07ed9c72-501c-4019-8460-02ae8a0078ee.png)